staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/metrics/metrics.go

```diff
@@ -130,7 +131,7 @@ var (
 			Help:           "The last time in seconds when a keyID was used.",
 			StabilityLevel: metrics.ALPHA,
 		},
-		[]string{"transformation_type", "provider_name", "key_id_hash"},
+		[]string{"transformation_type", "provider_name", "key_id_hash", "apiserver_id"},
```

Comment:
starting with these two metrics, any other metrics need the apiserver identity?

Reply:
You want all 3 metrics that include the `key_id_hash` label.

You also want all 3 of the `encryption_config_controller` metrics.